Formula/codestore.rb

```diff
@@ -0,0 +1,18 @@
+class Codestore < Formula
+  desc "Command-line interface of code.store"
+  homepage "https://code.store/"
+  url "https://s3.eu-central-1.amazonaws.com/s3.code.store/codestore-v1.2.12/codestore-v1.2.12.tar.gz"
+  sha256 "7474ac084e12cdd946b30e2eed7e5689cfbc656ff19e53f947141e67fa3c0d28"
+  license "MIT"
+  depends_on "node"
+
+  def install
+    inreplace "bin/codestore", /^CLIENT_HOME=/, "export CODESTORE_OCLIF_CLIENT_HOME=#{lib/"client"}\nCLIENT_HOME="
```

Comment:
It's this reported upstream?

Reply:
> It's this reported upstream?

Not sure to understand the comment, could you precise please? Just FYI, we are using Oclif to generate the formula https://oclif.io/docs/releasing#brew.

Reply:
https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff
> [...] it works without patching on all supported macOS releases [...]

If you need to rewrite parts of a file that should be reported upstream so it can be fixed.